target/linux/ramips/mt7621/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -0,0 +1,22 @@
+#!/bin/ash
```

Comment:
Is this file executable? Think this can be dropped, but haven't really checked ...